cache/in-memory/src/lib.rs

```diff
@@ -411,6 +411,16 @@ impl InMemoryCache {
         self.0.guild_emojis.get(&guild_id).map(|r| r.clone())
     }
 
+    /// Gets the set of integrations in a guild.
```

Comment:
This doc comment seems like its been switched around

Reply:
Good catch